common/agent.py

```diff
@@ -325,7 +326,7 @@ def gym_train(self):
                     s=state[0],
                     visual_s=state[1],
                     a=action,
-                    r=reward,
+                    r=reward.astype('float32'),
```

Comment:
refuse

Reply:
If the environment returns a reward in int,
there is an error in the calculations at least here:

replay_buffer.py
```
206 q [j] [3] + = data [3] * (self.gamma ** (_len - j))
```

Reply:
that's a bug, but I'd rather fix it in`gym_wrapper/wrapper_win.py` line 201. Let environment wrapper control anything about environment. I do not want any data type convertion occured in the section of training, because it's going to be messy to maintain this project.